### PR TITLE
Create Tigris databases

### PR DESCRIPTION
**Pull request by AlekSi**
Refs #782.

### CODE REVIEW

File: cmd/envtool/main.go [hunk 1]
```diff
@@ -216,14 +190,72 @@ func run(ctx context.Context, logger *zap.SugaredLogger) error {
...
+func run(ctx context.Context, logger *zap.SugaredLogger) error {
+	go debug.RunHandler(ctx, "127.0.0.1:8089", logger.Named("debug").Desugar())
+
+	if err := setupPostgres(ctx, logger); err != nil {
+		return err
+	}
+
+	if err := setupTigris(ctx, logger); err != nil {
+		return err
+	}
```
Comment (rumyantseva) [inline cmd/envtool/main.go:233]: Are those commands fast? Should we run them in goroutines with waitgroup to make startup a bit faster?

File: cmd/envtool/main.go [hunk 2]
```diff
@@ -216,14 +190,72 @@ func run(ctx context.Context, logger *zap.SugaredLogger) error {
...
+// setupTigris configures Tigris.
+func setupTigris(ctx context.Context, logger *zap.SugaredLogger) error {
+	logger = logger.Named("tigris")
+
+	if err := waitForPort(ctx, logger, 8081); err != nil {
+		return err
+	}
+
+	cfg := &config.Driver{
+		URL: "127.0.0.1:8081",
```
Comment (rumyantseva) [inline cmd/envtool/main.go:205]: Is this port `8081` a Tigris default port (like `5432` for Postgres)? The number `8081` is quite a common number for some sample apps, and I wonder if it's a good choice to take it.

**Review by rumyantseva — APPROVED** [review]
I have a couple of minor (non-blocking) concerns, see the comments.
In general, LGTM.

File: cmd/envtool/main.go (hunk 2 above)
Comment (AlekSi) [reply cmd/envtool/main.go:205]: It is, yeah:
* https://docs.tigrisdata.com/quickstart#startup-tigris-on-your-local-machine
* https://github.com/tigrisdata/tigris#building-with-docker-containers

We could map that port to some other port if there is a conflict, but let's wait for the conflict to happen

File: cmd/envtool/main.go (hunk 1 above)
Comment (AlekSi) [reply cmd/envtool/main.go:233]: The waitgroup was there before, with a bug :-) https://github.com/FerretDB/FerretDB/pull/788#pullrequestreview-1017562742

Right now we do very little in those functions, they are fast

Comment (rumyantseva) [reply cmd/envtool/main.go:233]: 😅

**Pull request merged**